lib/schema/json_shemata.ex

```diff
@@ -12,57 +12,57 @@ defmodule ApplicationRunner.JsonSchemata do
     GenServer.call(__MODULE__, {:get_schema_map, path})
   end
 
-  def load_raw_schema(schema, component_name) do
-    GenServer.call(__MODULE__, {:load_raw_schema, schema, component_name})
-  end
-
   def start_link(_) do
     GenServer.start_link(__MODULE__, %{}, name: __MODULE__)
   end
 
   # Server (callbacks)
   @impl true
   def init(_) do
-    root_json_directory = Application.app_dir(:application_runner, @component_api_directory)
-
-    relative_shemata_path =
-      Path.join(root_json_directory, "/**/*.schema.json")
-      |> Path.wildcard()
-      |> Enum.map(&Path.relative_to(&1, root_json_directory))
+    root_json_directory =
+      Application.app_dir(:application_runner, @component_api_directory) <>
+        "/component.schema.json"
```

Comment:
For easier maintainability, you can define the `"/component.schema.json"` in a const in top of the file near `@component_api_directory`
```suggestion
        @component_api_root_file
```

and in top of the file :
```elixir
@component_api_root_file "/component.schema.json"
```